src/components/DataViewer/SampleDataViewerWithButton.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+import React, { useState } from "react";
+import DataViewer from "@/components/DataViewer/DataViewer";
+// import PhoneIcon from "../Icons/PhoneIcon";
+import styled from "styled-components";
+
+interface ViewerProps {
```

Comment:
the type of data is also used in the dataviewer file. Maybe you can export the interface?

Reply:
DONE